src/playground/service.clj

```diff
@@ -73,6 +79,15 @@
     :enter (fn [context]
              (update context :request authentication-request session-auth-backend))}))
 
+(def authorization-interceptor
+  "throw unautherize by role"
+  {:name ::authorize
+   :enter (fn [context]
+            (let [role (get-in context [:request :session :identity :role])]
+              (if (= role models.user/admin-role)
+                context
+                (throw-unauthorized)
```

Comment:
`(throw-unauthorized)` will create HTTP 500, right?

Instead, look at the `interceptor-chain/terminate` pattern in this same file. Make sure to terminate witjh HTTP 403